modules/apps/headless/headless-builder/headless-builder-api/src/main/java/com/liferay/headless/builder/exception/InvalidBaseURLException.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.headless.builder.exception;
+
+import javax.ws.rs.BadRequestException;
+
+/**
+ * @author Sergio Jiménez del Coso
+ */
+public class InvalidBaseURLException extends BadRequestException {
```

Comment:
Do we need an extension of a `BadRequestException`? that is a JAX-RS exception. It is weird then that we need an ExceptionMapper

Reply:
@sergiojimcos will try to remove it.